api/docgen/examples.go

```diff
@@ -43,6 +45,56 @@ var exampleTxResponse string
 //go:embed "exampledata/resourceManagerStats.json"
 var exampleResourceMngrStats string
 
+var exampleBlob = `{
```

Comment:
This and big variable below can be 1) const (for clarity) but also 2) go:embed-ed

Reply:
done